documentation/en/user/source/configuration/storage.rst

```diff
@@ -272,6 +271,173 @@ GeoServer ``topp:states`` sample layer on a fictitious ``my-geowebcache-bucket``
+
+Using the MBTiles blob store will bring several benefits at the cost of some performance loss. We will be able to store our data with a significantly smaller number of files, which will help us avoid file system headaches and help us managing our data. In some cases the stored data will be more compacted reducing the size of the data on disk.
+
+When compared to the file blob store this store as two limitations:
+
+* This store will not integrate with disk quota, this is a consequence of using database files.
+* **This store cannot be shared among several GeoWebCache instances.**
+
+When managing this store files we should keep in mind that we are not dealing with simple files but databases files. When connections to a database file are open we should not delete, move or switch the associated file (or we may corrupt the databases files). Databases files can become fragmented after deleting an huge amount of data or after frequent inserts, updates or delete operations.
+
+File Path Templates
+````````````````````
+
+An MBTiles file will correspond to an SQLitle database file. In order to limit the amount of contention on each single database file we will allow users to decide the granularity of the databases files. When GeoWebCache needs to map a tile to a database file it will only look at the databases files paths, it will not take in account the MBTiles metadata (this is why we are able to handle others formats and projections).
+
+To configure the databases files granularity the user needs to provide a file path template. The default file path template for the MBTiles blob store is this one:
+
+.. code-block:: none
+
+  {grid}\{layer}\{format}\{params}\{z}\tiles_{x}_{y}.sqlite
+
+The provided files paths templates will always be considered relative to the root directory provided as a configuration option.
+
+In this case we are storing our tiles organized by the grid set id, the layer name, the tiles format, the parameters unique hash, the zoom level, column range and row range. The column and row range values are passed by configuration, by default those values are equal to 250.
+
+Follows an example of what the blob store root directory structure may look like when using the default file path template:
+
+.. code-block:: none
+
+  .
+  `-- EPSG_4326
+      `-- sf_restricted
+          `-- image_png
+              |-- f0023dc7bc347fee7a3a04dc797f2223f74e3448
+              |   `-- 14
+              |       `-- tiles_6750_12000.sqlite
+              `-- null
+                  |-- 10
+                  |   |-- tiles_0_0.sqlite
+                  |   |-- tiles_250_500.sqlite
+                  |   `-- tiles_250_750.sqlite
+                  `-- 9
+                      |-- tiles_0_0.sqlite
+                      `-- tiles_0_250.sqlite
+
+If no parameters were provided *null* string will be used. Is the responsibility of the user to define a file path template that will avoid collisions.
```

Comment:
I see your point and agree with you. Although, {layer}{grid}{format}{params} doesn't give enough granularity I will instead go for something like this: {layer}{grid}{format}{params}/{z}-{x}-{y}.sqlite with tiles ranges 250x250 which give us 62500 tiles per file. Does this sounds good ?